src/google.js

```diff
@@ -114,5 +114,19 @@ module.exports = (() => ({
     } catch (err) {
       throw new Error(`Unable to update contact ${err.message}`)
     }
+  },
+  deleteContact: function(resourceName) {
+    const peopleAPI = google.people({
+      version: 'v1',
+      auth: this.oAuth2Client
+    })
+
+    try {
+      return peopleAPI.people.deleteContact({
+        resourceName
+      })
+    } catch (err) {
+      throw new Error(`Unable to delete contact ${err.message}`)
+    }
```

Comment:
We'll never pass in the catch section since we're returning the promise and not the awaited / resolved promise, right?

Reply:
Correct, and we do this all over the file. i'll address this in a separate commit.

Reply:
Done in b5295cc01ac2107f676e55ec29717e5fb4b5bbda

Reply:
Before we expected to have 3 or 4 useful informations for the context. Now we just have "synchronisation is not ok : error.message" :( 

But thanks :)